lib/addons/p5.dom.js

```diff
@@ -419,45 +419,39 @@
    * @method createImg
    * @param  {String} src src path or url for image
    * @param  {String} [alt] alternate text to be used if image does not load
+   * @param  {String} [crossOrigin] <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes">crossOrigin property</a> of the &lt;img&gt; element; use either 'anonymous' or 'use-credentials' to retrieve the image with cross-origin access (for later use with &lt;canvas&gt;; by default, if no value or undefined is passed, CORS is not used
```

Comment:
This is a bit verbose for me, the more complete description of this functionality should probably be in the description of the function itself. The parameter description should ideally just say what this is and maybe the possible values for it.

Also you probably shoud add this to the other function signature below as well.

Reply:
Sure, I'll shorten the parameter description and add an example to illustrate its use.

The point of having a second method signature, IMO, was to allow people to provide a `successCallback` without specifying the optional `alt` parameter. In the same vein, they shouldn't have to specify `crossOrigin` if they just want to supply a `successCallback` in the second method signature, so I think it's better to leave it as is.

Reply:
The signature here is not just to document possible ways to use the function, it is also used by the unminified library to validate the parameters passed to the function by the user. If we want to illustrate a case where both `alt` and `crossOrigin` are omitted but `successCallback` is provided, we can add an additional signature instead.

Reply:
> If we want to illustrate a case where both `alt` and `crossOrigin` are omitted but `successCallback` is provided, we can add an additional signature instead.

I'm not sure if I've understood you correctly, but doesn' the second signature already do that?
https://github.com/processing/p5.js/blob/40b69e699a2f91637cbfb70e866e87def945edae/lib/addons/p5.dom.js#L430-L435

So, why do you think we need yet another signature?

Reply:
Let's keep discussion about which signature to include on the main thread. Treat this comment as about the verbosity only.